crt_portal/cts_forms/templates/forms/complaint_view/show/index.html

```diff
@@ -2,52 +2,52 @@
+      <div class="tablet:grid-col-4 grid-offset-1 padding-left-05">
+        <div class="display-flex">
+          <a class="outline-button outline-button--dark" href="/form/view{{ return_url_args }}">
+            <img src="{% static "img/intake-icons/ic_arrow_forward.svg" %}" class="icon">
+            Back to all
+          </a>
         </div>
+        <div class="display-flex flex-align-center details-id">
+          <h2 class="margin-right-205 margin-top-0 margin-bottom-0 backend-blue">
+            ID: {{ data.public_id }}
+          </h2>
+          <span class="status-tag status-{{data.status}} margin-top-05">
+            {{ data.status }}
+          </span>
+        </div>
+        <p>
+          <span class="details-date-label text-uppercase backend-blue">Received: </span>
+          <span class="backend-blue">{{ data.create_date|date:'g:i a F j, Y' }}</span>
+        </p>
+        <p>
+          <span class="details-date-label text-uppercase backend-blue">Last Updated: </span>
+          <span class="backend-blue">{{ data.modified_date|date:'g:i a F j, Y' }}</span>
+        </p>
       </div>
-      <div class="grid-row grid-gap-lg">
-        <div class="tablet:grid-col-4 grid-offset-1">
-          {% include 'forms/complaint_view/show/actions.html' with title="Actions" icon="img/intake-icons/ic_check-circle.svg" %}
-          <div class="activity-stream">
-            {% include 'forms/complaint_view/show/activity_stream.html' with title="Activity" icon="img/intake-icons/ic_activity.svg" %}
-          </div>
-          <div class="crt-portal-card">
-            <div class="crt-portal-card__content">
-              {% include 'forms/complaint_view/show/actions/comment_summary.html' with id_name="comment" is_summary=False note=comments.note button_text='Send' label='New comment' %}
-            </div>
-          </div>
+    </div>
+    <div class="grid-row grid-gap-lg">
+      <div class="tablet:grid-col-4 grid-offset-1">
+        {% include 'forms/complaint_view/show/actions.html' with title="Actions" icon="img/intake-icons/ic_check-circle.svg" %}
+        <div class="activity-stream">
+          {% include 'forms/complaint_view/show/activity_stream.html' with title="Activity" icon="img/intake-icons/ic_activity.svg" %}
         </div>
-        <div class="tablet:grid-col-6">
-          {% include 'forms/complaint_view/show/correspondent_info.html' with data=data %}
-          {% include 'forms/complaint_view/show/complaint_details.html' with data=data primaty_complaint=primary_complaint summary=summary %}
-          {% include 'forms/complaint_view/show/full_summary.html' with summary=data.violation_summary button_text='Send' %}
+        <div class="crt-portal-card">
+          <div class="crt-portal-card__content">
+            {% include 'forms/complaint_view/show/actions/comment_summary.html' with id_name="comment" is_summary=False note=comments.note button_text='Send' button_aria_label='send comment' label='New comment' %}
```

Comment:
I'm not seeing the `comments.note` come through as something I can populate the summary text input with. Gonna look into this tomorrow, because when you hit "edit" on an existing summary, you just get a blank box instead of the summary.

Reply:
This was one of the changes from the first PR and the edit comments PR. I was sending just the summary first, but then we needed summary and id and so then I sent the object. 

I can take a look.